qa/workunits/cephtool/test.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash -x
 
-source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
+if [ -z $CEPH_ROOT ]; then
+    source $(dirname $0)/../ceph-helpers.sh
+else
+    # launched by Makefile, so follow its instruction
+    source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
+fi
```

Comment:
I think that `dirname` should work for both cases. I thought @alimaredia had reverted this bit.